### DISCUSSION TITLE
Add Copy button for the path of file

### DISCUSSION THREAD

**Discussion by rakhadkz** [community/community#15708] [Pull Requests]
In the discussion, adding a copy button for the path of the file would be pretty conveniently

<img width="979" alt="Screen Shot 2022-04-28 at 06 01 45" src="https://user-images.githubusercontent.com/58052002/165650254-29464e39-287d-446f-89be-23dce8b2e74c.png">

**Comment by conorriches** [comment]
This would be so useful!

## Current behaviour
During PR feedback, in order to copy the path of a file with changes requested to open in my IDE, I have to carefully copy the path without it counting as a click on the header of the feedback so it doesn't collapse.

## Expected Behaviour
As a developer, I respond to feedback for code by going through the feedback items one by one, opening them in my IDE and then doing the thinking on responding to feedback there in a bigger picture environment.

Therefore having the ability to copy the path would make my life much easier.

## Why
* So its consistent with the code panels on the "files changed" view.
* So that I can respond to feedback with less friction, making my life easier, and developers happier people.
